From 0f4b4e825a469653f5b513d7f80f397722f49ae6 Mon Sep 17 00:00:00 2001
From: Pablo Sun <pablo.sun@mediatek.com>
Date: Mon, 30 Oct 2023 14:50:18 +0800
Subject: [PATCH] kmssink: Add mediatek into default driver list

The DRM driver "mediatek" has been supported by Linux mainline[1].
So add the driver name to the list of default drivers.

[1]: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/drivers/gpu/drm/mediatek/mtk_drm_drv.c

Signed-off-by: Pablo Sun <pablo.sun@mediatek.com>
---
 sys/kms/gstkmssink.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sys/kms/gstkmssink.c b/sys/kms/gstkmssink.c
index 788cefc06970..b56724e6356a 100644
--- a/sys/kms/gstkmssink.c
+++ b/sys/kms/gstkmssink.c
@@ -177,7 +177,7 @@ kms_open (gchar ** driver)
 {
   static const char *drivers[] = { "i915", "radeon", "nouveau", "vmwgfx",
     "exynos", "amdgpu", "imx-drm", "rockchip", "atmel-hlcdc", "msm",
-    "xlnx", "vc4", "meson", "sun4i-drm", "mxsfb-drm", "tegra",
+    "xlnx", "vc4", "meson", "sun4i-drm", "mxsfb-drm", "tegra", "mediatek",
     "xilinx_drm",               /* DEPRECATED. Replaced by xlnx */
   };
   int i, fd = -1;
-- 
2.18.0

